### PR TITLE
8321636: [testbug] Skip failing 3D lighting tests on macOS 14 / aarch64

### PR DESCRIPTION
**Pull request by kevinrushforth**
Until [JDK-8318985](https://bugs.openjdk.org/browse/JDK-8318985) is fixed, we will skip the 4 failing 3D light tests on Mac / aarch64 if the macOS version is 14, meaning they will not be run as part of our nightly headful test runs or developer test build.

---------
### Progress
- [x] Change must be properly reviewed (1 review required, with at least 1 [Reviewer](https://openjdk.org/bylaws#reviewer))
- [x] Change must not contain extraneous whitespace
- [x] Commit message must refer to an issue

### Issue
 * [JDK-8321636](https://bugs.openjdk.org/browse/JDK-8321636): [testbug] Skip failing 3D lighting tests on macOS 14 / aarch64 (**Bug** - P3)

### Reviewers
 * [Andy Goryachev](https://openjdk.org/census#angorya) (@andy-goryachev-oracle - **Reviewer**)

### Reviewing
<details><summary>Using <code>git</code></summary>

Checkout this PR locally: \
`$ git fetch https://git.openjdk.org/jfx.git pull/1303/head:pull/1303` \
`$ git checkout pull/1303`

Update a local copy of the PR: \
`$ git checkout pull/1303` \
`$ git pull https://git.openjdk.org/jfx.git pull/1303/head`

</details>
<details><summary>Using Skara CLI tools</summary>

Checkout this PR locally: \
`$ git pr checkout 1303`

View PR using the GUI difftool: \
`$ git pr show -t 1303`

</details>
<details><summary>Using diff file</summary>

Download this PR as a diff file: \
<a href="https://git.openjdk.org/jfx/pull/1303.diff">https://git.openjdk.org/jfx/pull/1303.diff</a>

</details>

### Webrev
[Link to Webrev Comment](https://git.openjdk.org/jfx/pull/1303#issuecomment-1850062752)

### CODE REVIEW

File: tests/system/src/test/java/test/robot/test3d/PointLightIlluminationTest.java [hunk 1]
```diff
@@ -66,9 +67,17 @@ public class PointLightIlluminationTest extends VisualTestBase {
     private static final double COLOR_TOLERANCE    = 0.07;
     private static volatile Scene testScene = null;
 
+    // Used to skip failing tests until JDK-8318985 is fixed
```
Comment (andy-goryachev-oracle) [inline tests/system/src/test/java/test/robot/test3d/PointLightIlluminationTest.java:70]: we probably should add a note in JDK-8318985 description so we won't forget to revert these changes.

**Review by andy-goryachev-oracle — APPROVED** [review]

File: tests/system/src/test/java/test/robot/test3d/PointLightIlluminationTest.java (hunk 1 above)
Comment (kevinrushforth) [reply tests/system/src/test/java/test/robot/test3d/PointLightIlluminationTest.java:70]: Good idea. I added a comment to that JBS bug.
